storage/src/test/java/tech/pegasys/teku/storage/server/VersionedDatabaseFactoryTest.java

```diff
@@ -37,6 +37,8 @@ public class VersionedDatabaseFactoryTest {
 
   private static final StateStorageMode DATA_STORAGE_MODE = PRUNE;
   public static final int MAX_KNOWN_NODE_CACHE_SIZE = 10_000;
+
+  public static final int DEFAULT_BLOCK_BATCH_SIZE = 25;
```

Comment:
would be easy to remove the configuration of batch size from this PR, may reduce noise, and if a follow up PR is to work this logic through, it'll all be changing anyway...

Reply:
reverted this cli change, will put back in in a subsequent PR.